src/main/java/httpserver/Main.java

```diff
@@ -24,15 +24,14 @@ public static void main(String[] args) throws Exception {
         String HEAD = "HEAD";
         String simple_get = "/simple_get";
 
-        Map<String, Map<String, Action>> routes = new HttpRoutes.Builder()
+        HttpRoutes routes = new HttpRoutes.Builder()
```

Comment:
nice!

Reply:
Damon pointed me to the builder functionality